<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QWidget" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>361</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Кредитный калькулятор</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sum">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sum">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_term">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_term">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Срок кредита</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_percent">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_percent">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_rubles">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_months">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>месяцев</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_percent">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>140</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_type">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тип ежемесячных платежей</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_annuity">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Аннуитентные</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_diff">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Дифференцированные</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_permonth">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ежемесячный платеж</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_charges">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Начисленные проценты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fullsum">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>400</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Долг + проценты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_permonth_value">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_charges_value">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_fullsum_value">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>400</y>
     <width>271</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_sum">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10000000000000000000000.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1000.000000000000000</double>
   </property>
   <property name="value">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_term">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>12</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_percent">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>12</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
